.gitignore

```diff
@@ -0,0 +1,20 @@
+*~
+*.autosave
+build*
+build.ninja
+clapack
+CMakeCache.txt
+CMakeFiles
+cmake_install.cmake
+hdf5
+install_manifest.txt
+Makefile
+matio
+msinttypes
+OpenMEEG
```

Comment:
No OpenMEEG needs to be removed from this list (this was true with the separate super-project, but is invalid now).